tests/PHPUnit/Unit/DeprecatedMethodsTest.php

```diff
@@ -86,20 +87,21 @@ public function test_deprecations()
         $this->assertDeprecatedMethodIsRemovedInPiwik4('Piwik\Columns\Dimension', 'getType');
     }
```

Comment:
could you revert the changes in coding style below?

Reply:
Sorry, my IDE did. I revert it. Waiting for Travis CI to check the changes I made.